code/download_data.sh

```diff
@@ -9,34 +9,15 @@ DATE_DIGITS=$(date +%Y%m%d)
 # Create URL
 URL=ftp://sidads.colorado.edu/DATASETS/NOAA/G02158/masked/$YEAR/${MONTH_NUM}_${MONTH_SHORT}/SNODAS_${DATE_DIGITS}.tar
 
+mkdir -p data
+pushd data
+
 # Download file from FTP server
 wget $URL
```

Comment:
If this returns anything but a 200 will the script exit with an error code?

Reply:
yes